### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by EvanBarbour3**
### Added
- Pre-approval documentation
- Dictionaries documentation

### CODE REVIEW

File: _includes/v4/installations.md [hunk 1]
```diff
@@ -274,3 +274,62 @@ Name | Required | Type | Description
...
+--- | --- | --- | ---
+`email` | Yes | string | Email address of the applicant
+`title` | Yes | string | Title. Accepted values: 'Mr', 'Mrs', 'Miss', 'Ms'. Case insensitive.
+`$.first_name` | Yes | string | First name
+`$.last_name` | Yes | string | Last name
+`$.date_of_birth` | Yes | string | Date Of Birth. Must be in an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) date format.
+`$.gender` | Yes | int | `1` for male, `2` for female
+`$.dependents` | Yes | int | The number of dependents the applicant has
+`$.marital_status` | Yes | int | A marital status described in [Get a Marital Status](#marital-statuses)
+`$.tel_home` | If `tel_mobile` is not provided | string | Home telephone number. MUST be a UK Home telephone number with the correct format (10-11 digits, e.g. 01611234567).
```
Comment (KrekoG) [inline _includes/v4/installations.md]: Should Home be capitalised?

File: _includes/v4/installations.md [hunk 2]
```diff
@@ -274,3 +274,62 @@ Name | Required | Type | Description
...
+`$.tel_mobile` | If `tel_home` is not provided | string | Mobile telephone number. MUST be a UK mobile number with the correct format (10-11 digits, e.g. 07123456789).
+`$.tel_employer` | Yes | string | Employer telephone number. MUST be a UK Home telephone number with the correct format (10-11 digits, e.g. 01611234567).
```
Comment (KrekoG) [inline _includes/v4/installations.md]: Can it really not be mobile? gash

File: _includes/v4/installations.md [hunk 3]
```diff
@@ -274,3 +274,62 @@ Name | Required | Type | Description
...
+`$.employment_status` | Yes | int | A marital status described in [Get an Employment Status](#employment-statuses)
```
Comment (KrekoG) [inline _includes/v4/installations.md]: employment status

File: _includes/v4/installations.md [hunk 4]
```diff
@@ -274,3 +274,62 @@ Name | Required | Type | Description
...
+
+> This API endpoint is a BETA endpoint and is subject to change based on feedback. This may not be the final variant of the API
+
+```
+POST {{ site.data.globals.api_prefix }}/installations/:installation/pre-approval
+```
+
+Retrieve the confidence level for an applicant in the format of a traffic lighting system.
+
+A response of `green` indicates that the applicant is of good credit worthiness, and from the information provided, nothing has been flagged on our decision checks.
```
Comment (srleach) [inline _includes/v4/installations.md]: I'd say 'with the information provided, nothing has been flagged in our decision checks'

File: _includes/v4/installations.md (hunk 2 above)
Comment (srleach) [reply _includes/v4/installations.md]: 'UK Landline telephone' rather than 'UK Home telephone' I think

File: _includes/v4/installations.md (hunk 3 above)
Comment (srleach) [reply _includes/v4/installations.md]: Yeah this looks wrong, though it may be the same doc. @EvanBarbour3 ?

**Comment by EvanBarbour3** [comment]
@KrekoG @srleach

File: _includes/v4/installations.md [hunk 5]
```diff
@@ -274,3 +274,62 @@ Name | Required | Type | Description
...
+`$.tel_mobile` | If `tel_home` is not provided | string | Mobile telephone number. MUST be a UK mobile number with the correct format (10-11 digits, e.g. 07123456789).
+`$.tel_employer` | Yes | string | Employer telephone number. MUST be a UK telephone number with the correct format (10-11 digits, e.g. 01611234567).
+`$.employment_status` | Yes | int | An employment status described in [Get an Employment Status](#employment-statuses)
+`$.employment_start_date` | Yes | date | Date of employment start. Must be in an [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) date format.
+`$.income` | Yes | int | Monthly income amount in pounds
+`$.debt` | Yes | int | Monthly debt repayments in pounds
+`$.bank_sort_code` | Yes | int | Bank sort code in the format of `xxxxxx`
+`$.bank_account_number` | Yes | int | Bank account number in the format of `xxxxxxxx`
+`$.addresses.*` | Yes | array | An array of address history. A minimum of 1 address should be provided
+`$.addresses.*.abode` | No | string | Flat, suite, floor details if present
```
Comment (KrekoG) [inline _includes/v4/installations.md]: if pressent = if applicable?

**Comment by wnowicki** [comment]
:OK

**Pull request merged**